asab/library/service.py

```diff
@@ -386,12 +386,15 @@ async def export(self, path="/", tenant=None, remove_path=False) -> typing.IO:
 		return fileobj
 
 
-	async def subscribe(self, paths):
+	async def subscribe(self, paths: typing.Union[str, typing.List[str]]) -> None:
 		"""
-		It subscribes to the changes in the library
+		Subscribe to changes for specified paths in the library.
 
-		:param paths: A list of absolute paths to subscribe to
+		:param paths: Either single path or list of paths to be subscribed. All the paths must be absolute (start with '/').
+		:type paths: list[str] | str
 		"""
+		if isinstance(paths, str):
+			paths = [paths]
```

Comment:
@eliska-n what do you say? Is user comfort more important than _do-not-mix-types-in-one-method_ principle? 😃